docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.2"
```

Comment:
`SHOULD` - Can this live in `.docker/` ?

Reply:
It can but then folks would have to go inside a folder starting with `.` to start a docker. This might be annoying for several windows programs like Windows. IMO its fine to clutter the root with this file.

Reply:
That's a good point.

🤔 

It sounds like we could get around that by [setting the `COMPOSE_FILE` environment variable](https://docs.docker.com/compose/reference/envvars/#compose_file):

`COMPOSE_FILE=.docker/docker-compose.yml:other/paths/the/user/wants`

We'd have to add that to the README, though, so it does add a bit of extra work for the user, and potential confusion.

I'm probably just being too pedantic, so the root folder is fine with me unless @coreymckrill would also prefer it in a subdir.

Reply:
I still lean towards not having to create an environment variable. I have not tested it out, but it looks like `COMPOSE_FILE` is used to specify multiple compose files, instead of defining a path. Path can simply be specified while executing `docker-compose` command. 

I am resolving this conversation now, but please re-open this if you feel we should rethink.